services/distribution/src/test/java/app/coronawarn/server/services/distribution/assembly/tracewarnings/structure/directory/file/TraceTimeIntervalWarningExportFileTest.java

```diff
@@ -61,6 +61,7 @@ void fromTraceTimeIntervalWarningsShouldReturnNewInstance() throws Exception {
     byte[] checksum = Files.readAllBytes(outputFile.toPath().resolve(FILE_NAME_CHECKSUM));
     byte[] checksum2 = Files.readAllBytes(outputFile2.toPath().resolve(FILE_NAME_CHECKSUM));
 
+    //then
```

Comment:
?

Reply:
I see an assert statement after this comment. So I would close this comment